src/main/java/pl/byczazagroda/trackexpensesappbackend/service/WalletService.java

```diff
@@ -15,4 +16,7 @@ public interface WalletService {
     WalletDTO createWallet(@Valid CreateWalletDTO createWalletDTO);
 
     List<WalletDTO> getWallets();
+
+    void deleteWalletById(@NotBlank(message = "Wallet id has to be greater than 0")
```

Comment:
Please remove @NotBlank annotation, and add @Min (1) ... and @Validated annotation above the class

The @NotBlank validation is against the String class, not the Long class.

Reply:
Done